app/Actions/WordPress/BatchAuthorizeEquipmentAction.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\Actions\WordPress;
+
+use App\Models\Customer;
+use App\Actions\WordPress\AuthorizeEquipmentAction;
+use Illuminate\Support\Facades\Log;
+
+class BatchAuthorizeEquipmentAction
```

Comment:
I would not do a batch action here. There's very little gained by it and now your logic is spread out across two classes in some interesting ways. Even getting rid of all of the "can you even train this person" stuff (as it's unnecessary), it's still harder to understand than the two loops below just enqueuing a whole bunch of `AuthorizeEquipmentAction` jobs.